editor/plugins/tiles/tile_set_atlas_source_editor.h

```diff
@@ -119,6 +119,8 @@ class TileSetAtlasSourceEditor : public HSplitContainer {
 
 	bool tile_set_changed_needs_update = false;
 
+	Vector<Ref<TileSetAtlasSource>> pendingAtlases;
```

Comment:
Please use snake_case for identifiers. You might see from surrounding code that we never use camelCase ;)